src/Catel.MVVM/Catel.MVVM.Shared/Collections/Extensions/SuspensionContextExtensions.cs

```diff
@@ -4,138 +4,232 @@
 // </copyright>
 // --------------------------------------------------------------------------------------------------------------------
 
-namespace Catel.Collections
-{
-    using System;
-    using System.Collections.Generic;
-    using System.Collections.Specialized;
 
+namespace Catel.Collections.Extensions
+{
```

Comment:
Your indentation seems to have gone to 2 spaces instead of 4. Please keep the 4.